tornado_rest_jsonapi/resource.py

```diff
@@ -45,10 +45,14 @@ def base_urlpath(self):
     def log(self):
         return app_log
 
-    def get_model_connector(self):
-        return self.model_connector(
-            application=self.application,
-            current_user=self.current_user)
+    def get_data_layer(self):
```

Comment:
because you don't have the current user in the init. Resource is a tornado web handler, and the current user is assigned asynchronously by prepare. Only during the execution of the handlers (get, post etc) you are guaranteed that prepare has been executed, and therefore the current user is defined. in init this is not the case.